examples/Makefile

```diff
@@ -27,19 +27,31 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
-
 EXAMPLES := adder/tests \
-	    endian_swapper/tests \
             axi_lite_slave/tests \
+            dff/tests \
```

Comment:
These tests fail in Travis with

` ValueError: Unable to accurately represent 10(us) with the simulator precision of 1e0`

Reply:
Fixed in #1132.